README.md

```diff
@@ -1,2 +1,178 @@
+
+# 📖 Leaderboard <a name="todo-list"></a>
+**"Leaderboard" is a project aimed at facilitating the testing and learning of API implementation in various projects.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JAVASCRIPT
+### Tech Stack <a name="tech-stack"></a>
+- Webpack
+
+<details>
+  <summary>Webpack</summary>
+  <ul>
+    <li><a href="https://webpack.js.org">Webpack</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Implementation of API to store items to the server]**
+- **[Intuitive UI that improves the visitng experience]**
+- **[A reload button to refresh your list]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://calebchris000.github.io/Hit-The-API/build/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ npm install webpack webpack-cli webpack-dev-server
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  git clone https://github.com/calebchris000/Hit-The-API.git
+  cd my-folder
+```
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  npm init -y
+  npm install
+  
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run build
+  or
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@githubhandle](https://github.com/calebchris000)
+- Twitter: [@twitterhandle](https://twitter.com/calebchris000)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/caleb-nwaizu-b815aa23b/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[The project would be interactive]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank the people involved in making this project possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] Please let's use a file in our root directory instead of an external link so that we would be able to customize the date to the current year. And the name of the developer. The updated year indicates the last date on which your license was renewed.